lib/phlex/html.rb

```diff
@@ -136,6 +136,14 @@ def call(...)
 				new(...).call
 			end
 			alias_method :render, :call
+
+			def new
+				if block_given?
+					raise ArgumentError, "You passed a block to #{name}.new. You probably meant to pass it to #{name}#call."
```

Comment:
This error message isn’t particularly helpful because most of the time, you should actually pass the block to the `render` method on another view.